Shared/qml/ContactReportSizePage.qml

```diff
@@ -18,7 +18,7 @@ import QtQuick 2.6
 import QtQuick.Controls 2.1
 import QtQuick.Controls.Material 2.1
 import QtQuick.Window 2.2
-import Esri.ArcGISRuntime.OpenSourceApps.DSA 1.1
+import Esri.ArcGISRuntime.OpenSourceApps.DSA 1.2
```

Comment:
What is this? It looks oddly similar to the app's version number, should this reflect `1.1.4`?

Reply:
Good catch, this should be 1.1 but was missed in the revert. QML version numbers go MAJOR.MINOR with no patch. I'll fix. 

In the long term it doesn't matter too much as it's in the process of being deprecated in Qt 6 though which I am thrilled about. QML file versioning only served to avoiding breaking changes which is resolved with better name resolution in Qt 6. If it were up to me I would force all QML versioning at 1.0; having QML file versions match app versions  is a maintenance nightmare where all files change needlessly with a patch.